packages/core/workflows-sdk/src/utils/composer/helpers/resolve-value.ts

```diff
@@ -41,9 +41,11 @@ export async function resolveValue(input, transactionContext) {
     }
 
     if (Array.isArray(inputTOUnwrap)) {
-      return await promiseAll(
+      const res = await promiseAll(
```

Comment:
nit: that can be reverted to just `return`

Reply:
oups debug session left over ahah